Examples/UIExplorer/js/CameraRollView.js

```diff
@@ -94,6 +94,7 @@ var CameraRollView = React.createClass({
         var imageStyle = [styles.image, {width: imageSize, height: imageSize}];
         return (
           <Image
+            key={asset}
```

Comment:
Hmm if it's used in the other example I'm OK with it although didn't know that keys could be objects.